### PR TITLE
Add environment specific ingressHost for lgy-iac-web

### PR DESCRIPTION
**Pull request by mark-butler-solirius**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RST-4469

### Change description ###

Add env specific iingressHost values for dev and stg

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: apps/dts-legacy/lgy-iac-web/dev.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: lgy-iac-web
+spec:
+  releaseName: lgy-iac-web
+  values:
+    java:
+      ingressHost: lgy-iac-web.dev.platform.hmcts.net
+      replicas: 1
```
Comment (timja) [inline apps/dts-legacy/lgy-iac-web/dev.yaml:10]: dev is not normally deployed via flux only via a pull request on Jenkins

File: apps/dts-legacy/lgy-iac-web/stg.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
```
Comment (timja) [inline apps/dts-legacy/lgy-iac-web/stg.yaml:1]: this likely needs to be referenced in the kustomization

**Review by timja — CHANGES_REQUESTED** [review]

File: apps/dts-legacy/lgy-iac-web/dev.yaml (hunk 1 above)
Comment (mark-butler-solirius) [reply apps/dts-legacy/lgy-iac-web/dev.yaml:10]: Thanks Tim - changing now

Comment (mark-butler-solirius) [reply apps/dts-legacy/lgy-iac-web/dev.yaml]: Pushed changes

**Review by timja — APPROVED** [review]

**Pull request merged**